mineinabyss-features/src/main/kotlin/com/mineinabyss/helpers/HelperFunctions.kt

```diff
@@ -130,3 +85,8 @@ fun Player.getGroups(): List<String> {
         ?.map { obj: InheritanceNode -> obj.groupName }?.toList() ?: emptyList()
 }
 
+class CreationFunctions {
```

Comment:
Ambiguous name

Reply:
suggestions? `BalanceCreationFunctions`? 🤡

Reply:
Factory is the general term here. Smth like `CoinFactory` might be broad enough while being concise